.github/workflows/stable.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         npm run archive
 
     - name: Release to Stable
-      uses: s0/git-publish-subdir-action@develop
+      uses: s0/git-publish-subdir-action@ac113f6bfe8896e85a373534242c949a7ea74c98 # develop
```

Comment:
Looks like there are tagged releases on this one and we could pick the latest to use there instead of develop?  https://github.com/s0/git-publish-subdir-action/releases

Reply:
I saw that but seems they haven't been putting out recent releases and they recommend pinning to a commit as well: https://github.com/s0/git-publish-subdir-action/issues/104.

My worry here is if we do go to the latest release (2.6.0 it looks like) because that release doesn't contain everything that is in `develop` and we've been relying on `develop` here, that may cause some breakage